backend/models/core/Entity.js

```diff
@@ -0,0 +1,111 @@
+import DB, { sha256, fieldsForSelect, fromAurora, sanitizeData, AURORA_TYPE } from '@backend/data_access/DB';
+import { log } from '@/lib/Logger';
+import UnauthorizedError from '@/components/errors/UnauthorizedError';
+import Session from '@backend/models/core/Session';
+import RecordNotFound from 'components/errors/RecordNotFound';
+
+class Entity {
+
+    static async entityTypes() {
+        const db = new DB();
+  
+        const sql = `SELECT entity_types.id, entity_types.name, entity_types.slug from entity_types`;
+         
+        const data = await db.exectuteStatement(sql, []);
+        
+        
+        return data.records.map(([
+            {longValue: entity_type_id},
+            {stringValue: entity_type_name},
+            {stringValue: entity_type_slug},
+          ]) => ({
+            entity_type_id, entity_type_name, entity_type_slug
+          })); 
+    }
+
+    static async value({entity_type_slug, row_id}) {
+      const db = new DB();
+
+      const sql = `SELECT entity_types.name, entity_types.slug, entities.slug, attributes.name, 
+                          attributes.type, \`values\`.row_id,
+      coalesce(\`values\`.value_string, \`values\`.value_long_string, \`values\`.value_integer, \`values\`.value_datetime, \`values\`.value_double) as value from \`values\`
+                          LEFT JOIN entity_types ON entity_types.id = \`values\`.entity_id
+                          LEFT JOIN entities ON entities.entity_type_id = \`values\`.entity_id
+                          LEFT JOIN attributes ON attributes.entity_id = \`values\`.entity_id
+      WHERE 
+          attributes.id =  \`values\`.attribute_id AND
+          entity_types.slug = :entity_type_slug AND 
+          \`values\`.row_id = :row_id
+          `;
+                
+      const data = await db.exectuteStatement(sql, [
+          {name: 'entity_type_slug', value:{stringValue: entity_type_slug}},
+          {name: 'row_id', value:{longValue: row_id}},
+      ]);
+      
+      return data.records.map(([
+          {stringValue: entity_type_name},
+          {stringValue: entity_type_slug},
+          {stringValue: entities_slug},
+          {stringValue: attributes_name},
+          {stringValue: attributes_type},
+          {longValue: row_id},
+          {stringValue: value},
+        ]) => ({
+          entity_type_name, entity_type_slug, entities_slug, attributes_name, attributes_type, row_id, value
+        })); 
```

Comment:
This is fine, and I reviewed that my initial solution is not any better.

Still, I envisioned the API to have the following format:
```
{  
  metadata:{
    total: 100,
    page: 1,
    per_page: 20,
  },
  data: [
  {
    id: 8, 
    slug: "how-to-run-shopify-cli-in-aws-cloud-9",
    title: "How to run Shopify CLI in AWS Cloud 9",
    body: "....",
    author: "Ardee Aram"
  }
}
```